deployment_logger.go

```diff
@@ -94,11 +99,32 @@ func (dlm DeploymentLogManager) WriteLog(log []byte) error {
 	return err
 }
 
+// check if there is enough space to store the logs
+func (dlm *DeploymentLogManager) haveEnoughSpaceForStoringLogs() bool {
+	var stat syscall.Statfs_t
+	syscall.Statfs(dlm.logLocation, &stat)
+
+	// Available blocks * size per block = available space in bytes
+	availableSpace := stat.Bavail * uint64(stat.Bsize)
+	if availableSpace < dlm.minLogSizeBytes {
+		fmt.Printf("Not enough space for storing logs: %v (required: %v)",
+			availableSpace, dlm.minLogSizeBytes)
+		fmt.Printf("Directory: %v", dlm.logLocation)
```

Comment:
Actually I shouldn't log at all. No space for logs anyway. Will just return error.